proto/nat.go

```diff
@@ -0,0 +1,28 @@
+package sonm
```

Comment:
Are you sure it's ok to add custom code to proto folder(just curious - looks ok at first sight)?

Reply:
It's okay.